roles/keycloak/defaults/main.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+keycloak_helm_release_name: keycloak
+keycloak_helm_chart_path: "../../charts/keycloak/"
+keycloak_helm_chart_ref: /usr/local/src/keycloak
+
+keycloak_helm_release_namespace: auth-system
+keycloak_helm_values: {}
+
+keycloak_host: "{{ undef('You must specify a Keycloak host using keycloak_host') }}"
+keycloak_ingress_annotations: {}
+keycloak_ingress_class_name: "{{ atmosphere_ingress_class_name }}"
+
+keycloak_admin_username: admin
+keycloak_admin_password: "{{ undef(hint='You must specify a Keycloak admin password') }}"
```

Comment:
I've found that this is not very useful to troubleshoot to be honest, when this alert comes, it's hard to know the exact missing variable.  We have two alternatives, we either include the variable name in this alert, or we replace it with a simple comment that will be easy for the user to understand:

```suggestion
# keycloak_admin_password: 
```